lib/strategy.js

```diff
@@ -61,7 +61,7 @@ function Strategy(options, verify) {
 
   OAuth2Strategy.call(this, options, verify);
   this.name = 'onshape';
-  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/session';
+  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/sessioninfo';
```

Comment:
Looked around … there does not seem to be any other references than in strategy.js (2 places … only one had the session call).

Note that the results returned from sessioninfo compare to session.

> On Jan 19, 2016, at 9:19 AM, Andrew Kimpton <notifications@github.com> wrote:
> 
> In lib/strategy.js <https://github.com/onshape/passport-onshape/pull/1#discussion_r50116345>:
> 
> > @@ -61,7 +61,7 @@ function Strategy(options, verify) {
> >  
> >    OAuth2Strategy.call(this, options, verify);
> >    this.name = 'onshape';
> > -  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/session';
> > +  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/sessioninfo';
> Can you look around the documentation in passport-onshape and see if there's a reference to options.userProfileURL that also needs to be updated?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/onshape/passport-onshape/pull/1/files#r50116345>.
>